src/App.js

```diff
@@ -1,19 +1,94 @@
-import React from 'react';
+import React, { Component } from 'react';
 import './App.css';
 
 import users from './api/users';
+import todos from './api/todos';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
+import { TodoList } from './components/TodoList/TodoList';
+import { Select } from './components/Select/Select';
+
+const userName = users.map(user => ({
```

Comment:
это не userName, userName это `Dmitry`
userName = 'Dmitry`

```suggestion
const usersNames = users.map(user => ({
```